src/main/java/com/leaseforlove/tagsmanagementservice/infraestructure/storage/migrations/config/ClientInitializerChange.java

```diff
@@ -0,0 +1,25 @@
+package com.leaseforlove.tagsmanagementservice.infraestructure.storage.migrations.config;
+
+import io.mongock.api.annotations.ChangeUnit;
+import io.mongock.api.annotations.Execution;
+import io.mongock.api.annotations.RollbackExecution;
+import org.springframework.data.mongodb.core.MongoTemplate;
+
+
+@ChangeUnit(id="client-initializer", order = "001", author = "mongock")
+public class ClientInitializerChange {
+
+    private final MongoTemplate mongoTemplate;
+
+    public ClientInitializerChange(MongoTemplate mongoTemplate) {
+        this.mongoTemplate = mongoTemplate;
+    }
+
+    @Execution
+    public void changeSet() {
+    }
+
+    @RollbackExecution
+    public void rollback() {
```

Comment:
vale a pena adicionar um rollback, se for criarmos a collection, se der algum problema, drop na collection que tentou criar.